### PR TITLE
changes in the student dashboard

### PR DESCRIPTION
**Pull request by manassolanki**
school-8000-desk

### CODE REVIEW

**Comment by rmehta** [comment]
leave + attendance should be a separate group... its significant.

**Comment by manassolanki** [comment]
New Student Dashboard will be like this
image


Added **Student Type** field in the program enrollment for **Boarding** and **Non-Boarding** students.
image

Also added **Walking** option in mode of transportation 
image

File: erpnext/schools/doctype/program_enrollment/program_enrollment.json [hunk 1]
```diff
@@ -143,27 +143,27 @@
...
+   "fieldtype": "Select", 
    "hidden": 0, 
    "ignore_user_permissions": 0, 
    "ignore_xss_filter": 0, 
    "in_filter": 0, 
    "in_global_search": 0, 
    "in_list_view": 0, 
    "in_standard_filter": 0, 
-   "label": "Enrollment Date", 
+   "label": "Student Type",
```
Comment (nabinhait) [inline erpnext/schools/doctype/program_enrollment/program_enrollment.json]: **Student Type** is not a good name, may be **Student Residency Type**

**Review by nabinhait — CHANGES_REQUESTED** [review]

File: erpnext/schools/doctype/program_enrollment/program_enrollment.json (hunk 1 above)
Comment (manassolanki) [inline erpnext/schools/doctype/program_enrollment/program_enrollment.json]: image

**Comment by manassolanki** [comment]
Student Type changed as 
image

**Pull request merged**